html/js/main.js

```diff
@@ -49,7 +49,10 @@ $(function () {
         FWRecorder.isReady = true;
         $uploadStatus.css({'color': '#000'});
         break;
-
+      case "show_premission_window_failed":
```

Comment:
As I understand, the only case when the `showPermissionWindow` function may throw an error is the case when there is no microphone connected to a device, right? Why can't we emit the existing `no_microphone_found` event?

There is another problem. Most probably, not only `showPermissionWindow`  function, but any other will throw the error you've discribed in #64, right? So, that's the second reason why the event should not be called `"show_premission_window_failed"`.